Update Documentation Structure and Content (#59)
emissions-api/emissions-api: docs/source/installation.rst

Installation
============

Linux (Fedora)
--------------

*Tested on Fedora 30*

The Emissions API relies on a few C libraries. The easiest way to get those as
well as their python bindings is to use dnf and get them from the system
repositories::

   %> dnf install python3-gdal python3-psycopg2 python3-pycurl python3-numpy

All other dependencies can easily be installed using pip. For this we use a

> virtualenvironment to avoid installing local dependencies globally in the

Reviewer: ```suggestion
virtual environment to avoid installing local dependencies globally in the
```

system. In the project folder, create and activate the virtual environment
allowing access to system packages as well::

   %> python3 -m venv --system-site-packages venv
   %> . ./venv/bin/activate

> Remember to always activate the invironment before you start working on the

Reviewer: ```suggestion
Remember to always use the environment while working on the
```

project since only this will give you access to the additional local libraries

> you install.

Reviewer: ```suggestion
you installed.
```
